Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -197,6 +197,15 @@ void WebProcessProxy::forWebPagesWithOrigin(PAL::SessionID sessionID, const Secu
     }
 }
 
+void WebProcessProxy::forWebPagesWithDataStore(const WebsiteDataStore& websiteDataStore, const Function<void(WebPageProxy&)>& callback)
+{
+    for (auto& page : globalPages()) {
+        if (!page || &page->websiteDataStore() != &websiteDataStore)
```

Comment:
I'd prefer if we compared sessionID's instead of data store pointers. It is identical at the moment but I find it clearer.